pkg/eve/runme.sh

```diff
@@ -59,6 +60,8 @@ Passing -v <local folder>:/out makes sure the file created is given most appropr
 
 -f fmt selects a packaging format: raw (default), qcow2, parallels, vdi and gcp are all valid options.
 
+-p disks selects disks to use for persist separated by comma (e.g. sda,sdb,sdc). May be used for installers.
```

Comment:
Why do we need this? We can already set eve_persist_disk and with these changes that can take a comma-separated list.

Reply:
With this change we can set it when we create installer (with docker run lfedge/eve…)

Reply:
But we can accomplish the same by putting it in grub.cfg in either the mounted config partition or in conf/grub.cfg and make.

If we want to be able to do this as part of the docker run it would make sense to add support for adding all of the options (with their existing names) which go into grub.cfg; that way I can also set the eve disk etc, and using the same names as opposed to unique option names for a subset.

Reply:
Instead of -p sda,sdb can we do -o eve_persist_disk=sda,sdb?
That way we can use -o for different grub options and also use it to specify the root disk.

Reply:
Done